requirements/doc.txt

```diff
@@ -190,7 +190,7 @@ six==1.16.0
     #   edx-sphinx-theme
 snowballstemmer==2.2.0
     # via sphinx
-sphinx==5.2.2
+sphinx==5.2.3
```

Comment:
No breaking change in the updated version https://www.sphinx-doc.org/en/master/changes.html#release-5-2-3-released-sep-30-2022